tests/testNRyield.cc

```diff
@@ -49,15 +55,22 @@ void testNRyield(G4double Emin, G4double Emax, const G4String& unit,
   if (!goodInput) ::exit(1);		// If anything failed, abort
 
   // Instantiate single instance of each of the named yield functions
-  const char* useNIEL[] = { "Lindhard", "LewinSmith", "Sarkis", "Impact" };
+  const char* useNIEL[] = {"Lindhard", "Empirical Lindhard", "LewinSmith", "Sarkis", "Impact" };
```

Comment:
Please remove the space from the name string.  In fact, just use "Empirical" here.